docs/source/yaml/world_schema.rst

```diff
@@ -24,15 +24,20 @@ The world schema looks as follows, where ``<angle brackets>`` are placeholders:
       locations: </path/to/location_data_file.yaml>
       objects: </path/to/object_data_file.yaml>
 
-   # Robots: For now, we only support a single robot
+   # Robots: Each robot contains basic properties, plus other add-ons such as path planners and grasp generators
    robots: 
-     - id: <value>
+     - name: <name>
```

Comment:
i'm curious how this even worked before!

Reply:
Well, two things:

1. This is just a doc page
2. If "name" is not a field, the world creation tool will simply name the robot based on its number e.g. `robot2`